tests/integration/security/fuzz/fuzz_test.go

```diff
@@ -45,44 +49,138 @@ spec:
 `
 )
 
+var (
+	// Known unsupported path parameter ("/bla;foo") normalization for Tomcat.
+	dotdotPwnIgnoreTomcat = []string{
+		"/../private/secret.html;index.html <- VULNERABLE!",
```

Comment:
is it smart enough to ignore permutations of similar bypass?

Reply:
I think not, I didn't find such options in it.